examples/tutorials/advance_model_config/01_microbatch.ipynb

```diff
@@ -0,0 +1,277 @@
+{
```

Comment:
<p>You can control <code>microbatch</code> from <code>train_model</code> action directlyto dynamically change ...:</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>